flow/embedded_views.h

```diff
@@ -250,6 +250,14 @@ class ExternalViewEmbedder {
 
   virtual bool SubmitFrame(GrContext* context);
 
+  // Caller should make sure to call this after |SubmitFrame|.
+  // Embedder that implements this method to do additional tasks after
+  // |SubmitFrame|. One of such examples is threading merging on iOS.
+  //
+  // After invoking this method, the current task on the TaskRunner should end
```

Comment:
It will be useful to document why this is the case as well.

Reply:
changed the whole paragraph a little bit, could you take a look again?